dockers/docker-database/Dockerfile.j2

```diff
@@ -11,21 +11,22 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update
 
 # Install redis-server
+# TODO: Require direct bullseye-security installation after the OS release version upgraded
```

Comment:
I suggest just "direct bullseye-security installation after the OS release version upgraded"

Reply:
Bullseye-security is part of the `sources.list` file, so just saying `apt install redis-server` should install the latest version (whether that be from bullseye-security or bullseye-updates). Given that, does bullseye-security need to be explicitly mentioned?

Reply:
@qiluo-msft I agree with @saiarcot895, since we have upgraded to bullseye, so we can use apt installation directly.